designer/apps/core/wagtailadmin/views.py

```diff
@@ -14,16 +15,18 @@ class SiteCreationView(TemplateView):
     """
     template_name = 'wagtailadmin/site_creation.html'
 
-    @method_decorator(login_required)
     def dispatch(self, request, *args, **kwargs):
-        if not request.user.is_staff:
+        if not request.user.has_perm('add_site'):
```

Comment:
Who has the "add_site" permission today? Moderators?

Reply:
Currently just super users. Once it becomes self service, it'll be moderators (non-devs who manage sites). Easily addable to any group via the Group Settings.